ddtrace/contrib/internal/subprocess/patch.py

```diff
@@ -342,29 +482,40 @@ def _traced_ossystem(module, pin, wrapped, instance, args, kwargs):
             span.set_tag_str(COMMANDS.COMPONENT, "os")
             ret = wrapped(*args, **kwargs)
             span.set_tag_str(COMMANDS.EXIT_CODE, str(ret))
-        return ret
-    except Exception:  # noqa:E722
-        log.debug("Could not trace subprocess execution for os.system", exc_info=True)
+            return ret
+    else:
         return wrapped(*args, **kwargs)
 
 
 @trace_utils.with_traced_module
 def _traced_fork(module, pin, wrapped, instance, args, kwargs):
+    """Traced wrapper for os.fork function.
+
+    Note:
+        Only instruments when AAP or IAST is enabled.
+        Creates spans with fork operation details.
+    """
+    ret = wrapped(*args, **kwargs)
     if not (asm_config._asm_enabled or asm_config._iast_enabled):
-        return wrapped(*args, **kwargs)
+        return ret
     try:
         with pin.tracer.trace(COMMANDS.SPAN_NAME, resource="fork", span_type=SpanTypes.SYSTEM) as span:
```

Comment:
what is the point of this span? it seems this would be basically measuring the time it takes to create a span and set 2 tags?

would this now create a span in both the parent and the child?